src/components/TodoList/TodoList.tsx

```diff
@@ -1,12 +1,40 @@
-/* eslint-disable */
 import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { useAppSelector } from '../../app/hooks';
+import { useDispatch } from 'react-redux';
+import { setCurrentTodo } from '../../features/currentTodo';
+
+type Props = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  const dispatch = useDispatch();
+  const filter = useAppSelector(state => state.filter);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const filteredTodos = todos.filter(todo => {
+    switch (filter.status) {
+      case 'completed':
+        return todo.completed;
+      case 'active':
+        return !todo.completed;
+      default:
+        return todos;
```

Comment:
The return statement inside the default case of the switch should return `true` to include all todos, not `todos`. This is likely a mistake and could lead to unexpected behavior.